coalib/processes/Processing.py

```diff
@@ -598,8 +597,7 @@ def process_queues(processes,
     result_files = set()
     ignore_ranges = list(yield_ignore_ranges(file_dict))
 
-    # One process is the logger thread (if not in debug mode)
-    while local_processes > (1 if not debug else 0):
+    while local_processes > 0:
```

Comment:
@Makman2 I changed this conditions for processes as I thought we correct. All the tests are passing but coala stops while execution AutoPEP8 section, can you please take a look and correct me if there is something wrong with these conditions?

Reply:
I am pretty sure that removing the logger_thread and changing these conditions is breaking my builds.

Reply:
I'm not able to understand why is this happening as I have removed all logger_thread dependencies .

Reply:
Phew that's bad then... I don't know how to fix it, there are rivets and bolts everywhere, and you've just seen it's nearly impossible to maintain. I can just advise you to debug further, sorry :/